CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.6.0
+
+Features:
+* Upgrade to Go 1.14 [#347](https://github.com/datastax/cass-operator/issues/347)
+* Add support for specifying additional PersistentVolumeClaims [#327](https://github.com/datastax/cass-operator/issues/327)
+* Add support for specify rack labels [#292](https://github.com/datastax/cass-operator/issues/292)
+
+Bug fixes:
+* Retry decommission to prevent cluster from getting stuck in decommission state [#356](https://github.com/datastax/cass-operator/pull/356)
+* Make the busybox image configurable [#339](https://github.com/datastax/cass-operator/pull/339)
```

Comment:
as I recall this maybe needed to be retitled, and we just added a tag to the busybox image, instead of `latest`